jetbrains-core/src/software/aws/toolkits/jetbrains/services/s3/objectActions/DeleteObjectAction.kt

```diff
@@ -45,7 +45,7 @@ private fun deleteNodes(project: Project, treeTable: S3TreeTable, nodes: List<S3
     if (response != Messages.OK) {
         S3Telemetry.deleteObject(project, Result.Cancelled)
     } else {
-        GlobalScope.launch {
+        ApplicationThreadPoolScope("DeleteObjectAction").launch {
```

Comment:
feels weird that the scope isn't attached to the action

Reply:
Yeah so this function is actually called when there is not a copy of the action as well, so it can't just move into the class (the first thing I tried)